staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1692,3 +1695,40 @@ enum ContainerEventType {
     // Container deleted
     CONTAINER_DELETED_EVENT = 3;
 }
+
+message ListPodSandboxMetricsRequest {} 
+
+message ListPodSandboxMetricsResponse {
+    repeated PodSandboxMetrics pod_metrics = 1;
+}
+
+message PodSandboxMetrics {
+    string pod_sandbox_id = 1;
+    repeated Metric metrics = 2;
+    repeated ContainerMetrics container_metrics = 3;
+}
+
+message ContainerMetrics {
+    string container_id = 1;
+    repeated Metric metrics = 2;
+}
+
+message Metric {
+    string name = 1;
+    string help = 2;
```

Comment:
are we sure we want help here? This is a bit unfortunate because it means we need to send help text for every single metric when help is the same for the same metric "name". It seems like it will increase the rpc payload size alot due to the duplication (since the metric is per container). 

Maybe we can omit this for now and figure out best way to include this later?